modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -1297,7 +1297,7 @@ TEST_P(PhaseTest, AccuracyTest)
     // Comparison //////////////////////////////////////////////////////////////
     // FIXME: use a comparison functor instead (after enabling OpenCL)
     {
-        EXPECT_EQ(0, cvtest::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
+        EXPECT_LT(cvtest::norm(out_mat_ocv, out_mat_gapi, NORM_INF), 4e-6);
```

Comment:
hmm in google tests' `EXPECT(a,b)` `a` is a golden reference usually, does this mean it should be `EXPECT_GT(4e-6, norm..))` ?

Reply:
This rule is for `_EQ` only due to output messages (LT,LE,GT,GE doesn't require that)

Reply:
Why not `EXPECT_FLOAT_EQ(0, cvtest::norm())` / `EXPECT_NEAR()` ? 4e-6 looks like a magic number to me.